client/src/components/Overview/CartForm.jsx

```diff
@@ -1,16 +1,44 @@
-import React, { useContext } from 'react';
-import ProductContext from '../../context';
+import React, { useState } from 'react';
+
+export default function CartForm({ currentStyle }) {
+  const [selectedSku, setSelectedSku] = useState('default');
+  const [quantity, setQuantity] = useState([]);
+
+  const skuIds = Object.keys(currentStyle.skus);
+
+  function handleSizeSelection(event) {
+    const eventSku = event.target.value;
+    setSelectedSku(eventSku);
+    const quantityArray = [];
+    for (let i = 1; i <= currentStyle.skus[eventSku].quantity; i += 1) {
+      if (i <= 15) {
```

Comment:
What if you had a variable like maxQuantity, which is the minimum of 15 and the available quantity - that way you wouldn't need the if block inside the for loop, and you wouldn't loop more than you had to.

Reply:
Good idea! Changed!